test/blackbox/LatencyBudgetReqRepHelloWorldRequester.hpp

```diff
@@ -0,0 +1,145 @@
+// Copyright 2016 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Comment:
Can this file be merged with `ReqRepHelloWorldRequester` adding getters and setters, and functionality for latency purposes?

```suggestion
// Copyright 2019 Proyectos y Sistemas de Mantenimiento SL (eProsima).
```

Reply:
Done

Reply:
Changed